news/2020-05-02-beatmapping-world-championship-2020.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "osu! Beatmapping World Championship #2"
+date: 2020-05-02 12:00:00 +0000
+---
+
+Welcome to the second edition of the osu! Beatmapping World Championship! Check out the changes to the contest format as well as everything else to look forward to!
+
+![](https://i.imgur.com/oqHbVX8.jpg)
```

Comment:
the banner should be put in ``wiki/shared/news/banners`` as an image instead i guess

Reply:
Yup, fixed it